tests/test_cmab.py

```
@@ -1249,3 +1252,88 @@ def test_random_seed_propagates_to_bnn(
+        action_ids=action_ids,
+        n_features=n_features,
+        backbone_hidden_dims=[16, 8],
+        embedding_dim=6,
+        n_svi_steps=15,
+        random_seed=3,
+    )
+    assert isinstance(cmab, CmabBernoulli)
+    assert isinstance(cmab.actions_manager.meta_model, BackboneHeadsMetaModel)
+
+    context = rng.normal(size=(12, n_features))
+    actions, probs, weighted_sums = cmab.predict(context=context)
```

Ruff (0.15.17)

[warning] Unpacked variable `weighted_sums` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)